Hud/HUDElements/TipText.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Common;
+using OpenTK.Mathematics;
+
+namespace Hud.HUDElements;
+public class TipText : IHudElement
+{
+    public bool Visible { get; set; } = true;
+
+    private readonly float _size = 0.03f;
+
+    public void Dispose()
+    {
+        // nothing to dispose
+    }
+
+    public void Render(Shader shader)
+    {
+        shader.SetVector4("color", Vector4.One);
+
+        Printer.RenderStringTopLeft(shader, "C to enter the car", _size, 0, 0);
```

Comment:
Maybe make it more generic so that it can be used for any type of pop up text

Reply:
flipping the car is a cool idea considering that rollovers are pretty common, I'll add it